docs/setup-android.md

```diff
@@ -107,6 +109,40 @@ protected String getJSMainModuleName() {
 }
 ```
 
+If you are integrating React Native into existing application please do the following:
```

Comment:
I'd prefer to say here `If you are integrating React Native into existing native application please do the following steps:`